File: codigos algoritmicos python/ejercicio25, 26, 27, 28 .py
print("MENU")
print("1. Saludar")
print("2. Mostrar fecha")
print("3. Salir")

numero = int(input("Ingrese un numero: "))

if numero >= 1:

    if numero == 1:
        nombre = input("Ingrese su nombre: ")
        print("Hola", nombre)

    else:

        if numero == 2:
            print("La fecha de hoy es 25/06/2024")

        else:

            if numero == 3:
                salir = input("¿Desea salir? (s/n): ")

                if salir.lower() == "s":
                    print("¡Hasta luego!")
                else:
                    print("Continuando con el programa...")

            else:
                print("Opción no válida")

File: codigos algoritmicos javascript/ejercicio25, 26, 27, 28.js
console.log("MENU DE OPCIONES");
console.log("1. saludar");
console.log("2. mostrar fecha");
console.log("3. salir");

let opciones = 2;

if (opciones == 1) {
    console.log("Hola, bienvenido al programa");
} else {

    if (opciones == 2) {
        let fecha = new Date();
        console.log("La fecha de hoy es 25/06/2024");
    } else {

        if (opciones == 3) {
            console.log("Saliendo del programa...");
        } else {
            console.log("Opción no válida");
        }

    }

}
